samples/M4/configure.ac

```diff
@@ -0,0 +1,143 @@
+#                                               -*- Autoconf -*-
```

Comment:
Also, because of the Emacs modeline, maybe one of:

- Add an `autoconf` alias to M4.
- Add a separate Autoconf language with `group: M4`.